lms/routes.py

```diff
@@ -12,6 +12,7 @@ def includeme(config):
     config.add_route("reports", "/reports")
 
     config.add_route("canvas.v11.config", "/config_xml")
+    config.add_route("canvas.v13.config", "/canvas/1.3/config")
```

Comment:
If we followed the file layout then the two variations (1.1 and 1.3) are in the same file, which might suggest the hierarchy is `canvas/config/1.3`.

On the other hand I've seen conventions where the it's something like `<api_version>/<path>` before which is a bit more like this. Is that what you were going for?